src/hooks/useMarketsData.ts

```diff
@@ -240,15 +246,22 @@ export const useMarketsData = ({
     return filtered;
   }, [markets, searchFilter, filter]);
 
+  const hasPredictionMarkets = markets.some((market) =>
+    Object.values(PREDICTION_MARKET).includes(market.id)
+  );
+
   const marketFilters = useMemo(
-    () => [
-      MarketFilters.ALL,
-      MarketFilters.NEW,
-      ...objectKeys(MARKET_FILTER_OPTIONS).filter((marketFilter) =>
-        markets.some((market) => market.tags?.some((tag) => tag === marketFilter))
-      ),
-    ],
-    [markets]
+    () =>
+      [
+        MarketFilters.ALL,
+        MarketFilters.NEW,
+        testFlags.pml && MarketFilters.LAUNCHABLE,
+        ...objectKeys(MARKET_FILTER_OPTIONS).filter((marketFilter) =>
+          markets.some((market) => market.tags?.some((tag) => tag === marketFilter))
+        ),
+        hasPredictionMarkets && MarketFilters.PREDICTION_MARKET,
+      ].filter(isTruthy),
```

Comment:
.filter(Boolean) is easier xD

Reply:
<img width="519" alt="Screen Shot 2024-10-21 at 9 14 10 AM" src="https://github.com/user-attachments/assets/2f9841cc-cc66-46ce-994b-4ec8b72bce9c">

Reply:
filter(isTruthy) or filter(isPresent) reads much nicer toooo